download_controller.py

```
@@ -0,0 +1,86 @@
+    async def get_filename_from_header(self, content_disposition):
+        if not content_disposition:
+            return None
+        fname = re.findall('filename="?([^"]+)"?', content_disposition)
+        if len(fname) == 0:
+            return None
+        return fname[0]
+
+    def get_download_links(self, comic_article_link: str) -> str:
+        headers = {"User-Agent": "Mozzilla/5.0"}
+
+        response = requests.get(comic_article_link, headers)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)